src/focus-visible.js

```diff
@@ -276,7 +276,7 @@ function applyFocusVisiblePolyfill(scope) {
 // It is important to wrap all references to global window and document in
 // these checks to support server-side rendering use cases
 // @see https://github.com/WICG/focus-visible/issues/199
-if (typeof window !== 'undefined' && typeof document !== 'undefined') {
+if (typeof window !== 'undefined' && typeof document !== 'undefined' && !window.applyFocusVisiblePolyfill) {
```

Comment:
Just checking for `applyFocusVisiblePolyfill` is sufficiently safe as the function is always run right after it's assignment into the global Window scope